### PR TITLE
[-] FO : fixed bug on payment module, check if defined $order object

### PR DESCRIPTION
**Pull request by kelu95**

### CODE REVIEW

**Comment by xBorderie** [comment]
Rework of #3035. Thanks @kelu95 !

File: classes/PaymentModule.php [hunk 1]
```diff
@@ -406,7 +406,7 @@ public function validateOrder($id_cart, $id_order_state, $amount_paid, $payment_
                     $transaction_id = null;
                 }
 
-                if (!$order->addOrderPayment($amount_paid, null, $transaction_id)) {
+                if (!isset($order) || !$order->addOrderPayment($amount_paid, null, $transaction_id)) {
```
Comment (kpodemski) [inline classes/PaymentModule.php]: shouldn't be `if(!Validate::isLoadedObject($order)` ??

Comment (kelu95) [inline classes/PaymentModule.php]: I will add this check to, but need to kkep the isset in reference to this bug http://forge.prestashop.com/browse/PSCSX-5576

**Comment by vTerenti** [comment]
QA OK

**Pull request merged**